vector/sidecar_server.py

```diff
@@ -15,21 +15,28 @@
 
 """
 
-import os
+import signal
 import subprocess
+import sys
 import time
 from pathlib import Path
 
-ppid = os.getppid()
-print(f"{ppid=}", flush=True)
+
+def signal_handler(sig):
```

Comment:
This fixes the `signal_handler() takes 1 positional argument but 2 were given` error:
```suggestion
def signal_handler(sig, _):
```